### DISCUSSION TITLE
文法及單字救星-Grammarly!

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#140] [General]
Grammarly存在於許多的網站中，它自動幫忙檢查拼字,標點符號,文法及語句等等。雖然使用者可以免費使用，但它還是賺進了高達130億美元的資金。Grammarly有各種工具集的寫作工具，也可以運用在寫信方面，給予符合時態和邏輯的建議; 一個特別的點是:當使用者想給給不同的人時，它會給予不同的建議，朋友或者是老闆等等，可以選擇不同的風格來回信，寫信。另外，除了免費的程式，它還提供了個人版的Grammarly Premium和企業版的Grammarly Business兩種模式，給使用者更多進階型功能及回饋。
以後寫論文或是寫報告需要用到英文時，Grammarly絕對是大家的好幫手!

文章來源:https://technews.tw/2021/12/05/grammarly-valuation/

**Comment by chia-kai-chang** [comment]
這家真的好用，寫論文的好幫手

**Comment by pocper1** [comment]
我也有使用Grammarly! 可以辨別錯字，還蠻好用的

**Comment by bonny123865** [comment]
Grammarly超棒!因為高中英文都忘得差不多，所以打文章拼字很常出錯，都會幫我標記出來，不會讓我打到懷疑自己！

**Comment by briankuo123** [comment]
印象中在上大一英文時，我的同學也有被老師要求要用這款。可能是以前改到太多文法或拼音錯誤受不了了吧哈哈。

**Reply by chia-kai-chang** [reply]
有點尷尬~~~
就像是送朋友廁所清潔劑的概念(嫌朋友的廁所髒)~

**Comment by wh20502** [comment]
這項技術我覺得非常好用，因為很常看到報告中有錯或是簡報中有錯字，就會下意識的被認為不專業，所以我覺得這個報告必備。

**Comment by ghost** [comment]
這個軟體聽朋友之前介紹過 有的時候寫太快的確文法會錯誤 能夠抓出錯誤也是一大便利產物呢

**Comment by handsomeniko** [comment]
滿分工具，我的大專生科技部計畫靠他拯救過

**Comment by Shiba010** [comment]
grammarly真的非常方便，我也有在用！

**Comment by ayakasanma** [comment]
在YOUTUBE還沒買會員的時候常常聽到廣告，蠻好笑的XD

**Comment by Lynn12378** [comment]
最近常常youtube上看到，感覺還蠻好用的
